drools-ansible-rulebook-integration-main/src/test/java/org/drools/ansible/rulebook/integration/main/SlownessTest.java

```diff
@@ -32,8 +32,12 @@ static public void afterClass() {
     @Test
     public void testOnceAfter() {
         try {
-            SlowAutomaticPseudoClock.enable(11000, 4000);
-            ExecuteResult result = Main.execute("56_once_after.json");// it has "delay_warning_threshold":"2 seconds"
+            // <drools.delay.warning.threshold>2</drools.delay.warning.threshold> configured in pom.xml.
+            // But explicitly set it here in case of running the test from IDE.
+            System.setProperty("drools.delay.warning.threshold", "2");
```

Comment:
This property should be unset at the end of the test.

Reply:
Thanks, updated.